core/controllers/editor_test.py

```diff
@@ -146,8 +146,6 @@ def test_editor_page(self):
         response = self.get_html_response('/create/0')
         self.assertIn('Help others learn new things.', response.body)
         self.assert_can_edit(response.body)
-        self.assertIn('Stats', response.body)
-        self.assertIn('History', response.body)
```

Comment:
@kevinlee12 @seanlip These are the lines I had to remove to let the tests pass. (SInce it can no longer find 'Stats' or 'History' as they are directives now)

Reply:
I think that's OK. Thanks for checking.